1-exercises/C-object-methods/exercise-2.js

```diff
@@ -15,10 +15,10 @@ let myPets = {
   rabbit: "Bugs",
 };
 
-let petNames =
-  /* todo */
+let petNames = Object.values(myPets);
+/* todo */
```

Comment:
Well done Bahare 😊.
I think the execise require you to return the values in an array.